docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -526,6 +580,37 @@ public class PersonResource {
 NOTE: Be careful to use the `@Transactional` annotation on the operations that modify the database,
 you can add the annotation at the class level for simplicity purpose.
 
+To make it easier to showcase some capabilities of Hibernate ORM with Panache on Quarkus, some test data should be inserted into the database by adding the following content to a new file named src/main/resources/import.sql:
```

Comment:
Maybe it's worth mentioning that this will only work when running in dev or test mode. See https://quarkus.io/guides/hibernate-orm#dev-mode